<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
      <item>
       <layout class="QVBoxLayout" name="customerInfoPanel" stretch="0,0,0,0">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="emailLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="addressLabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="textFormat">
           <enum>Qt::TextFormat::PlainText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ageLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableAccounts">
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="withdrawButton">
        <property name="text">
         <string>Withdraw</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="depositButton">
        <property name="text">
         <string>Deposit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addAccountButton">
        <property name="text">
         <string>Open Account</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeAccountButton">
        <property name="text">
         <string>Close Account</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="replaceCardButton">
        <property name="text">
         <string>Replace Card</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showTransactionsButton">
        <property name="text">
         <string>Transactions History</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportTransactionsButton">
        <property name="text">
         <string>ExportTransactions</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="transactionList">
      <property name="resizeMode">
       <enum>QListView::ResizeMode::Fixed</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="cardListWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
